decidim-core/app/presenters/decidim/notification_presenter.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Decidim
+  #
+  # Decorator for notifications.
+  #
+  class NotificationPresenter < SimpleDelegator
+    TIME_RANGES = {
+      1.minute => :seconds_time,
+      1.hour => :minutes_time,
+      1.day => :hours_time,
+      1.month => :days_time,
+      1.year => :long_time
+    }.freeze
+
+    def created_at_in_words
```

Comment:
I know it's still a draft PR, but there is any reason for not using the [time_ago_in_words](https://apidock.com/rails/v6.0.0/ActionView/Helpers/DateHelper/time_ago_in_words) helper?

Reply:
I've decided using this custom version because I can be more accurate with the time in words displayed. Anyway, I'd test if the time_ago_in_words helper could fit the requirements.

Reply:
e.g. with this helper we get `less than a minute` when the seconds go from 30 to 59. Or `about an hour` when minutes are from 45 to 59. If that's ok, I can simplify the code using the helper.

Reply:
Yes, that's OK!! I prefer to have that behavior but simplifying and making it consistent with other parts of the application.